src/plugins/unified_histogram/public/chart/histogram.tsx

```diff
@@ -189,6 +189,7 @@ export function Histogram({
           disabledActions={disabledActions}
           onFilter={onFilter}
           onBrushEnd={onBrushEnd}
+          withDefaultActions
```

Comment:
Does this work as intended? It doesn't seem to be passed down from anywhere.

Reply:
yes, this was a mistake. I needed pass `withDefaultActions` passed by parents here. It is now corrected.